.travis.yml

```diff
@@ -67,12 +70,14 @@ jobs:
       os: osx
       before_script:
         - travis_wait rustup install $(cat rust-toolchain)
+        - travis_wait rustup component add rustfmt-preview
     - stage: cerrors
       os: linux  # No need to run this everywhere
       before_script:
         - sudo apt update
         - sudo apt install -y texinfo libgif-dev libxpm-dev
         - travis_wait rustup install $(cat rust-toolchain)
+        - travis_wait rustup component add rustfmt-preview
```

Comment:
Why have you added rustfmt to these stages of the build?

Reply:
The bindings get run through rustfmt. Technically I suppose it's optional, but if we ever have to actually look at the bindings to figure out a build error, then we will definitely want them to be formatted nicely.